composer.json

```diff
@@ -17,7 +17,9 @@
         "symfony/yaml": "^3.1",
         "symfony/class-loader": "^3.1",
         "doctrine/annotations": "^1.2",
+        "ircmaxell/random-lib": "^1.2",
```

Comment:
I have deprecated this lib, in favour of the PHP7 backwards lib random_compat. Is there some reason why you need this?